UpdateExpiredCgn/table.ts

```diff
@@ -10,24 +12,33 @@ import {
   TableEntry
 } from "../utils/table_storage";
 
+const ExpiredCgnRowKeyType = t.interface({
+  activationDate: Timestamp,
+  expirationDate: Timestamp,
+  fiscalCode: FiscalCode
+});
+
+export type ExpiredCgnRowKeyType = t.TypeOf<typeof ExpiredCgnRowKeyType>;
+
 /**
  * Do something with the user hash extracted from the table entry
  */
-const withFiscalCodeFromEntry = (f: (s: FiscalCode) => void) => (
+const withExpiredCgnRowFromEntry = (f: (s: ExpiredCgnRowKeyType) => void) => (
```

Comment:
If the above is applied
```suggestion
const withExpiredCgnRowFromEntry = (f: (s: ExpiredCgnRowKey) => void) => (
```